lib/pdf_generation/participants_pdf.rb

```diff
@@ -0,0 +1,87 @@
+class ParticipantsPDF
+  include Prawn::View
+  Prawn::Font::AFM.hide_m17n_warning = true #consider adding TTF font
+ 
+  def self.generate(event)
+    self.new(event).create.render
+  end
+
+  def initialize(event)
+    @event = event
+    @document = Prawn::Document.new(page_size: 'A4')
+    @participants_count = @event.participants.count
+    @participants_allergic_application_letters = []
+    @participants_vegan_application_letters = []
+    @participants_vegeterian_application_letters = []
+    @participants_omnivorous_application_letters = []
+    @event.participants.each do |p|
+
+      letter = ApplicationLetter.where(user_id: p.id, event_id: @event.id).first
+      if !letter.nil?
+        @participants_allergic_application_letters.push(letter) if letter && letter.allergic
+        @participants_vegeterian_application_letters.push(letter) if letter && letter.vegeterian
+        @participants_vegan_application_letters.push(letter) if letter && letter.vegan
+        @participants_omnivorous_application_letters.push(letter) if letter && !letter.vegan && !letter.vegeterian
+      end
+
+    end
+  end
+
+  def create   
+    create_summary
+    self
+  end
+
+  private
+    def create_summary
+      text t("events.participants.print_title", title: @event.name), size: 20
+      move_down 20
+      text @event.date_ranges[0].to_s, size: 10
+      move_down 20
+      text t("events.participants.print_summary", number: @event.participants.count), size: 15
+      
+      if @participants_omnivorous_application_letters.count > 0
+        text t("events.participants.print_summary_omnivorous", number: @participants_omnivorous_application_letters.count), size: 15
+        @participants_omnivorous_application_letters.each do |p|
+          text p.user.name, size: 10
+        end
```

Comment:
According to Coveralls, this block isn't covered by any tests. I know it's trivial but... think of those gorgeous 100%.